### PR TITLE
Re-jigged the landing and getting started pages

### PR DESCRIPTION
**Pull request by mattpocock**
This is an incremental improvement towards a better top-of-the-funnel experience. It removes the `/docs/getting-started` page, and merges it into `/docs`. It takes the more verbose, 'why' stuff and makes it a core concept - 'Why Turborepo'.

It also prettifies the front page of the docs significantly:

image

### CODE REVIEW

File: docs/components/Feature.tsx [hunk 1]
```diff
@@ -11,23 +11,63 @@ const DetailedFeatureInner = (props: { feature: FeatureProps["feature"] }) => {
   const { Icon, name, description } = props.feature;
   return (
     <>
+      <div className="bg-circle-pink"></div>
```
[the remark this answers is not in the document]
Comment (mattpocock) [reply docs/components/Feature.tsx:14]: I don't think these BGCircles will reasonably be used in another component. They're tied to the size and shape of the cards. If they are, we can do the refactor then.

File: docs/components/Feature.tsx [hunk 2]
```diff
@@ -11,23 +11,63 @@ const DetailedFeatureInner = (props: { feature: FeatureProps["feature"] }) => {
...
+          {name}
+        </h3>
       </div>
       <div>
-        <p className="mt-2 text-base font-medium text-gray-500 dark:text-gray-400">
+        <p className="mt-2 text-base font-medium leading-7 text-gray-500 dark:text-gray-400">
           {description}
         </p>
       </div>
+      <style jsx global>{`
```
[the remark this answers is not in the document]
Comment (mehulkar) [reply docs/components/Feature.tsx:32]: minor +1 for component. The styles make a good case for scoping to a separate component. Components aren't just about reuse, they're also about isolation.

File: docs/pages/docs/core-concepts/why-turborepo.mdx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Why Turborepo?
+
+## The problem
+
+Monorepos have many advantages - but **they struggle to scale**. Each workspace you add has its own test suite, its own linting and its own build process. A single monorepo might have **hundreds of tasks to perform**.
```
Comment (mehulkar) [inline docs/pages/docs/core-concepts/why-turborepo.mdx:5]:
```suggestion
Monorepos have many advantages - but **they struggle to scale**. Each workspace has its own test suite, its own linting and its own build process. A single monorepo might have **hundreds of tasks to execute**.
```

"execute" for consistency "executing tasks" is a recurring phrase everywhere to really drill in by repetition 🏋🏾

**Review by mehulkar — APPROVED** [review]
Minor suggestions, but copy looks great to me, nice work!

**Pull request merged**